server/views/partial/one-signal.ejs

```diff
@@ -1,5 +1,11 @@
-<script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" async></script>
 <script>
+  (() => setTimeout(() => {
+    const myScript = document.createElement("script");
+    myScript.setAttribute("src", "https://cdn.onesignal.com/sdks/OneSignalSDK.js");
```

Comment:
What happens if oneSignal is not used?

Reply:
https://github.com/quintype/malibu-advanced/blob/onesignal-refactor/app/server/handlers/render-layout.js#L75
https://github.com/quintype/malibu-advanced/blob/onesignal-refactor/views/pages/layout.ejs#L43